README.zh-CN.md

```diff
@@ -61,27 +61,569 @@ Apache EventMesh提供了许多功能来帮助用户实现他们的目标，以
+```
+tar -xvzf apache-eventmesh-1.10.0-source.tar.gz
+cd apache-eventmesh-1.10.0-src/
+```
+
+使用 Gradle 构建源代码。
+```
+gradle clean dist
+```
+
+编辑`eventmesh.properties`以更改EventMesh Runtime的配置（如 TCP 端口、客户端黑名单）。
+```
+cd dist
+vim conf/eventmesh.properties
+```
+
+2.构建并加载插件  
+Apache EventMesh引入了 SPI 机制，使 EventMesh 能够在运行时发现并加载插件。有两种方式安装插件:  
+- Gradle依赖项: 在`eventmesh-starter/build.gradle`中将插件声明为构建依赖项。  
+```
+dependencies {
+   implementation project(":eventmesh-runtime")
+
+    // 示例： 加载 RocketMQ 插件
+   implementation project(":eventmesh-connectors:eventmesh-connector-rocketmq")
+}
+```
+
+- 插件目录: EventMesh会根据`eventmesh.properties`加载`dist/plugin`目录中的插件。Gradle的`installPlugin`任务会构建插件并将其移动到`dist/plugin`目录中。  
+```
+gradle installPlugin
+```
+
+3.启动Runtime   
+
+执行`start.sh`脚本启动EventMesh Runtime服务器。
+```
+bash bin/start.sh
+```
+
+查看输出日志：  
+```
+tail -f logs/eventmesh.out
+```
+
+#### 远程部署
+
+在[EventMesh download](https://eventmesh.apache.org/download/)页面选择所需要版本的Binary Distribution进行下载,您将获得`apache-eventmesh-1.10.0-bin.tar.gz`。   
+1.下载:  
+```
+# 解压
+tar -xvzf apache-eventmesh-1.10.0-bin.tar.gz
+cd apache-eventmesh-1.10.0
+```
+
+2.部署  
+
+编辑`eventmesh.properties`以更改EventMesh Runtime的配置（如 TCP 端口、客户端黑名单）。
+```
+vim conf/eventmesh.properties
+```
+
+执行`start.sh`脚本启动EventMesh Runtime服务器。  
+```
+bash bin/start.sh
+```
+如果看到`EventMeshTCPServer[port=10000] started....`, 则说明设置成功。   
+
+查看输出日志:  
+```
+cd /root/apache-eventmesh-1.10.0/logs
+tail -f eventmesh.out
+```
+
+停止:  
+```
+bash bin/stop.sh
+```
+
+#### Docker部署EventMesh Runtime
+
+准备:   
+- 建议使用64位的linux系统。
+- 请预先安装Docker Engine。Docker的安装过程可以参考[docker官方文档](https://docs.docker.com/engine/install/)。
+- 建议掌握基础的docker概念和命令行，例如注册中心、挂载等等。不过这不是必须的，因为本次操作所需的命令都已为您列出。
+- 若您选择非standalone模式，请确保[RocketMQ](https://rocketmq.apache.org/docs/quickStart/01quickstart/)已成功启动并且可以使用ip地址访问到；若您选择standalone模式，则无需启动RocketMQ 。
+
+1.获取EventMesh镜像  
+首先，你可以打开一个命令行，并且使用下面的`pull`命令从[Docker Hub](https://hub.docker.com)中下载最新发布的[EventMesh](https://hub.docker.com/r/apache/eventmesh)。   
+```
+sudo docker pull apache/eventmesh:v1.10.0
+```
+
+您可以使用以下命令列出并查看本地已有的镜像。
+```
+sudo docker images
+```
+
+如果终端显示如下所示的镜像信息，则说明 EventMesh 镜像已经成功下载到本地。
+```
+$ sudo docker images
+REPOSITORY            TAG       IMAGE ID       CREATED         SIZE
+apache/eventmesh     v1.10.0    6e2964599c78   10 days ago     937MB
+```
+
+2.创建配置文件:  
+在根据EventMesh镜像运行对应容器之前，你需要创建两个配置文件，分别是:`eventmesh.properties`和`rocketmq-client.properties`。  
+首先，你需要使用下面的命令创建这两个文件。  
+```
+sudo mkdir -p /data/eventmesh/rocketmq/conf
+cd /data/eventmesh/rocketmq/conf
+sudo touch eventmesh.properties
+sudo touch rocketmq-client.properties
+```
+
+3.配置`eventmesh.properties`  
+
+这个配置文件中包含 EventMesh 运行时环境和集成进来的其他插件所需的参数。  
+使用下面的`vim`命令编辑`eventmesh.properties`。  
+```
+sudo vim eventmesh.properties
+```
+你可以直接将 GitHub 仓库中的对应[配置文件](https://github.com/apache/eventmesh/blob/1.10.0-prepare/eventmesh-runtime/conf/eventmesh.properties)中的内容复制过来。  
```

Comment:
I think it is unnecessary to include the 'Deployment Guide' in the project README. The README is meant to help people gain a better understanding of our project. Instead, we can provide a URL that links to the Deployment Guide.

Furthermore, please note that the current document is outdated and no longer practical. You can find an updated version at https://github.com/apache/eventmesh-site/pull/170.

Reply:
@Pil0tXia I think it is still necessary, the previous connection is to jump to the official website, there are a lot of outdated, not updated in time, which is not conducive to rapid understanding, because the user must first want to run the project deployment. If the home page contains this guide, it can be updated in time, and the command error can be corrected in time. Of course, this is only a simple introduction guide, more detailed official documents are needed to interpret. What do you think ?

Reply:
Perhaps can simplify it, or remove some of the deployment guidelines, leaving only the local deployment ? What are your good suggestions ?

Reply:
In fact, only third-party dependency libraries have deployment and usage instructions in the README.md because they are small in size and do not require a documentation website. Moreover, they are typically known for their simplicity, and straightforward syntax tends to attract people interested in the project.

However, in the messaging system field, as well as in big data and middleware domains, the majority of users are not aware of what the project does until they visit its GitHub homepage. The README.md is the first page they see, and it is crucial to clearly explain the middleware's purpose, capabilities, and where to use to help users assess the project's value and potential.

The deployment process of EventMesh is not simple, and it is not our strongest suit, at least for now. Lengthy, non-navigable, and non-expandable documentation quickly loses the interest of users and developers.

There is no need to worry that "providing document hyperlinks will make users overlook the project's usage instructions." Other large Apache projects follow the same approach. We can supplement our README document with key features and provide hyperlinks to the documentation website. Once EventMesh supports more automated one-click deployment in the future, we can add it to the README. I believe this strikes the perfect balance between simplicity and comprehensiveness at present.

Reply:
Our WeChat official account and Bilibili have also published numerous successful implementation cases and practical experiences of companies. I believe that these are valuable resources and experiences, but they are rarely read. They should be included in the "Additional Readings" section of our README.

For example:

- [吉利汽车如何基于EventMesh落地事件集成平台，助力业务高效集成](https://mp.weixin.qq.com/s/BLHbIq5KyjnhoVlmw9_DBw)
- [解决SaaS组合式应用集成标准化问题--EventMesh在华为的实践应用](https://www.bilibili.com/video/BV1cA4y1d7uw/)